workflow/rocoto/gfs_tasks.py

```diff
@@ -110,15 +110,19 @@ def waveinit(self):
 
         resources = self.get_resource('waveinit')
         dependencies = None
+        cycledef = None
         if self.app_config.mode in ['cycled']:
             deps = []
-            dep_dict = {'type': 'task', 'name': f'{self.cdump}prep'}
-            deps.append(rocoto.add_dependency(dep_dict))
-            dep_dict = {'type': 'cycleexist', 'condition': 'not', 'offset': '-06:00:00'}
-            deps.append(rocoto.add_dependency(dep_dict))
+# TODO: Cleanup for PR
+#            dep_dict = {'type': 'task', 'name': f'{self.cdump}prep'}
+#            deps.append(rocoto.add_dependency(dep_dict))
+#            dep_dict = {'type': 'cycleexist', 'condition': 'not', 'offset': '-06:00:00'}
+#            deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
Why is this commented out?

Reply:
Using this code block results in:

```
   <dependency>
		<or>
			<taskdep task="gdasprep"/>
			<not><cycleexistdep cycle_offset="-06:00:00"/></not>
		</or>
	</dependency>
```

and

```
 <dependency>
		<or>
			<taskdep task="gfsprep"/>
			<not><cycleexistdep cycle_offset="-06:00:00"/></not>
		</or>
	</dependency>
```

for the `waveprep` tasks. Is that what you expect?

Reply:
For `waveinit`, not `waveprep`, yes.

Reply:
Sorry, I think that I meant `waveinit` here.